public/javascript/controllers/user-ctrl.js

```diff
@@ -33,6 +33,20 @@ angular.module('harvestv2')
                 return (pass1 == pass2);
             };
 
+
+		CurrentUserFactory.query($routeParams, function(currentuser) {
+   			$routeParams.id = currentuser._id;
+			UserFactory.show($routeParams, function(user) {
+                        	$scope.user = user;
+                        	var tempdate = new Date($scope.user.us_user_creation_date );
+                        	$scope.user.us_user_creation_date  = tempdate.getFullYear()+'-' + (tempdate.getMonth()+1) + '-'+tempdate.getDate();
```

Comment:
@thinkking angular supports various versions of user readable date.  Checkout [Angular Date Documentation](https://docs.angularjs.org/api/ng/filter/date)